### DISCUSSION TITLE
copilot not working within PyCharm

### DISCUSSION THREAD

**Discussion by bwagner** [community/community#41826] [Copilot Conversations]
### copilot not working within PyCharm

Bug

I signed up for github copilot and was accepted.

Installed the PyCharm plugin following [these instructions](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide).

I choose Tool>Github Copilot>Login to Github.
I am provided with a Device Code which I enter on the github site which responds with a page where I can authorize, which I do, which is ackowledged with "Congratulations, you're all set! Your device is now connected."
Back in PyCharm the status line displays "Github Copilot: No access to Github Copilot found."
<img width="548" alt="Screenshot 2022-12-15 at 14 21 12" src="https://user-images.githubusercontent.com/447049/207869646-5defb127-03b2-4fd4-9317-c53dc0320956.png">
<img width="600" alt="Screenshot 2022-12-15 at 14 23 19" src="https://user-images.githubusercontent.com/447049/207870036-bf39916b-617c-4f40-ab13-3f8d1924526f.png">

PyCharm 2022.3 (Professional Edition)
Build #PY-223.7571.203, built on November 30, 2022
Licensed to Bernhard Wagner
Subscription is active until November 4, 2023.
For educational use only.
Runtime version: 17.0.5+1-b653.14 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.6.2
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Non-Bundled Plugins:
    IdeaVIM (2.0.0)
    com.github.copilot (1.1.37.2188)
    zielu.gittoolbox (213.10.3)
    sourcery.pycharm-plugin (1.0.2)
    org.nik.presentation-assistant (1.0.10)

Thanks

**Comment by Akash1134** [comment]
Hi @bwagner could you please reload the PyCharm plugin then restart Copilot and see if that helps? Thanks!
